site/components/src/main/java/uk/nhs/digital/ps/components/PublicationComponent.java

```diff
@@ -51,7 +53,8 @@ public void doBeforeRender(final HstRequest request, final HstResponse response)
             index.add(KEY_FACTS_ID);
         }
 
-        if (isNotBlank(publication.getAdministrativeSources())) {
+        // TODO remove publication check in DW-1285
```

Comment:
Did you mean to leave a todo in the code?

Reply:
Yes, just a reminder to remove this in the 'clean-up' ticket